boards/common/stm32/Kconfig.g4

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) 2020 Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+menu "STM32 G4 clock configuration"
+    depends on CPU_FAM_G4
+
+choice
+bool "Clock source selection"
+default USE_CLOCK_PLL
+
+config USE_CLOCK_PLL
+    bool "PLL"
+
+config USE_CLOCK_HSE
+    bool "Direct High frequency external oscillator (HSE)"
+    depends on BOARD_HAS_HSE
+
+config USE_CLOCK_HSI
```

Comment:
Is this missing the configuration of `CONFIG_CLOCK_HSISYS_DIV`?

Reply:
There's no HSISYS division factor on G4.

Reply:
What about this? https://github.com/RIOT-OS/RIOT/blob/df5e347947168a4548ca639aabca4337543cba43/boards/common/stm32/include/g4/cfg_clock_default.h#L85-L87

Reply:
removed :)